masonite/commands/MiddlewareCommand.py

```diff
@@ -0,0 +1,36 @@
+""" New Middleware Command """
+import os
+
+from masonite.view import View
+from masonite.app import App
+from masonite.helpers.filesystem import make_directory
+
+from cleo import Command
+
+
+class MiddlewareCommand(Command):
```

Comment:
This should look more like this file https://github.com/MasoniteFramework/core/blob/develop/masonite/commands/ModelCommand.py

Reply:
🤔 no lol. Maybe we should add that way in this pr?

Just need to take a look at the BaseScaffoldCommand and maybe add something like:

```
...
postfix='Middleware'
```

and then we can add that to scaffolding commands to postfix stuff.

Reply:
you'll need to add the logic to append file names in that `BaseScaffoldCommand`.

Reply:
How about `suffix` true.

```
...
suffix = True
```